### PR TITLE
Updated de (German) translation

### PR DESCRIPTION
**Pull request by TalviT**
Commit made via Stringlate

### CODE REVIEW

File: res/values-de/strings.xml [hunk 1]
```diff
@@ -110,7 +150,7 @@
 	<string name="show_comment">Kommentar zeigen</string>
 	<string name="hide_comment">Kommentar verbergen</string>
 	<string name="links_menu_item">Links</string>
-	<string name="share_comments">Kommentare teilen</string>
+	<string name ="share_comments">Kommentare teilen</string>
```
Comment (zagaberoo) [inline res/values-de/strings.xml]: Erroneous space between 'name' and '='

**Comment by TalviT** [comment]
Hm. I don't know where these spaces come from... maybe i should open an issue for the Stinglate app^^
Thanks for marking.

**Comment by zagaberoo** [comment]
Definitely sounds like it could be a Stringlate bug.

Thanks for contributing!

**Comment by TalviT** [comment]
After opening a new issue at Stringlate's repository, I just noticed that it was no bug in Stringlate. It looks as if the space character is already in your original '[strings.xml](https://github.com/zagaberoo/diode/blob/master/res/values/strings.xml#L184)' file (in line 184).

**Comment by zagaberoo** [comment]
Aha, that makes perfect sense.  I should have known :)

**Comment by zagaberoo** [comment]
There are new strings to be translated if you're up for it: https://github.com/zagaberoo/diode/compare/5d43716a2a7f049a910a04c15fa53a4eb35dea9e...zagaberoo:master#diff-4691bafd7791e471b30c492e6f43d65b

**Pull request merged**